### DISCUSSION TITLE
List of tested cards

### DISCUSSION THREAD

**Discussion by jeremyherbert** [makinako/OpenFIPS201#53] [Ideas]
Hi,

Just as a suggestion, it would be useful to have a list of tested cards so that users could refer to it before purchasing a card. Also, listing the amount of memory needed (both EEPROM and transient) may help to decide which cards are appropriate. 

Thanks,
Jeremy

**Comment by makinako** [comment]
Hi @jeremyherbert, this is a good idea and high time it was included!

I've started this on the wiki here:
https://openfips201.atlassian.net/wiki/spaces/OD/pages/950373/Applet+Requirements#2.2-Tested-Platforms

And for the next release I'll run it against any 3.0.4+ cards we have and capture memory requirements. If you have any to contribute feel free to add them to this discussion or comment on the wiki page.
